src/Application/Migrations/22_view_serviceable_tickets.sql

```diff
@@ -34,40 +30,46 @@ CREATE OR REPLACE VIEW view_serviceable_tickets AS
 	LEFT JOIN
 		tbl_project_encoding_profile pep ON pep.project_id = pj.id AND pep.encoding_profile_version_id = t.encoding_profile_version_id
 	LEFT JOIN
-		tbl_ticket_state state ON state.ticket_type = t.ticket_type AND state.ticket_state = t.ticket_state
-	LEFT JOIN
-		tbl_ticket_state wantedstate ON wantedstate.ticket_type = 'encoding' AND wantedstate.ticket_state = pj.dependent_ticket_trigger_state
+		tbl_ticket_state configured_trigger_state ON
+			configured_trigger_state.ticket_type = 'encoding' AND
```

Comment:
I guess ```configured_trigger_state.ticket_type = t.ticket_type``` would do the same, but it always be _encoding_ as long we don't allow other ticket types as ```encoding``` to have a _encoding_profile_version_id_ set.
I'm ok with this. It's just I like to avoid as much hard-coding as possible without performance impact.